test/util_test.go

```diff
@@ -51,27 +32,3 @@ func generateECChain(t *testing.T, tsg *sim.TipSetGenerator) gpbft.ECChain {
 	require.NoError(t, err)
 	return chain
 }
-
-// repeatInParallel repeats target for the given number of repetitions.
-// Set F3_TEST_REPETITION_PARALLELISM=1 to run repetitions sequentially.
-// See repetitionParallelism.
-func repeatInParallel(t *testing.T, repetitions int, target func(t *testing.T, repetition int)) {
-	// When no parallelism is requested, run repetitions sequentially so their logs are readable.
-	if repetitionParallelism <= 1 {
-		for i := 0; i <= repetitions; i++ {
-			t.Log("repetition", i)
-			target(t, i)
-		}
```

Comment:
For development, how will one in the future target a specific test instance (specific participant count and random seed) in order to isolate its logs, use debugger, and quickly repeat it?

Reply:
Based on later comments it seems this is no longer an issue; marking as resolved. please let me know if I have missed anything.